### ISSUE TITLE
share information of inside

### ISSUE THREAD

**Issue by ehsansis0** [Manouchehri/Feitian-MultiPass-FIDO-U2F#2]
Hi Dear

thanks very much for share of your information about multipass. could you please share some pictures about board and used chip from manufacture. i rally like to get some information about circuit and physical board.

my email address: ehsansis0@gmail.com

Thanks very much
Ehsan Farahani Asil

**Comment by Manouchehri** [comment]
I still use this device sometimes, so ideally I’d like to avoid cracking it open.

What information were you hoping to find? The chip has already been identified.

**Comment by ehsansis0** [comment]
Thanks for your response. I found some pictures from inside but pictures doesn't have good quality. Could you tell me about model of all chip used for this device.?
Thanks

**Comment by ehsansis0** [comment]
Thanks for your response. I found some pictures from inside but pictures doesn't have good quality. Could you tell me about model of all chip used for this device.?
Thanks

**Comment by ehsansis0** [comment]
?

**Comment by Manouchehri** [comment]
It's already listed at https://github.com/Manouchehri/Feitian-MultiPass-FIDO-U2F#useful-nfc-info. See `Title: NXP Semiconductors SmartMX (P5CD081 or P5CD041) tag with NXP JCOP v2.4.2 R1`.

**Comment by ehsansis0** [comment]
yes, i check it. could you please send some pictures from board and inside chip?

**Comment by Manouchehri** [comment]
I've long stopped using this device; still interested in pictures? I can probably find it in a bin somewhere and disassemble it for you.

**Comment by Maksym-0xp** [comment]
@Manouchehri 
If you ended up with some pictures, then it would be interesting to see them. Im still using mine so don't want to do permenent damage just yet :)

**Comment by Manouchehri** [comment]
Blast from the past!

I think I gave mine away years ago. 😅 If I find it somewhere, I'll crack it open.
